### ISSUE TITLE
[New Rule] replace `zip(d.keys(), d.values())` with `d.items()`

### ISSUE THREAD

**Issue by GideonBear** [MartinThoma/flake8-simplify#161]
## Explanation

It's shorter and more pythonic

## Example

```python
# Bad
zip(d.keys(), d.values())

# Good
d.items()
```
If this is accepted, I would be interested in implementing this.
Thanks for the great flake8 plugin!
